tilecloud/__init__.py

```diff
@@ -726,69 +726,85 @@ def load(name):  # pragma: no cover
+            )
+            from tilecloud.lib.memcached import MemcachedClient  # pylint: disable=import-outside-toplevel
 
             server, template = name[12:].split("/", 1)
             host, port = server.split(":", 1)
             client = MemcachedClient(host, int(port))
             return MemcachedTileStore(client, TemplateTileLayout(template))
         if name.startswith("s3://"):
-            from tilecloud.layout.template import TemplateTileLayout
-            from tilecloud.store.s3 import S3TileStore
+            from tilecloud.layout.template import (  # pylint: disable=import-outside-toplevel
+                TemplateTileLayout,
+            )
+            from tilecloud.store.s3 import S3TileStore  # pylint: disable=import-outside-toplevel
 
             bucket, template = name[5:].split("/", 1)
             return S3TileStore(bucket, TemplateTileLayout(template))
         if name.startswith("sqs://"):
-            from tilecloud.store.sqs import SQSTileStore
-            import boto.sqs
-            from boto.sqs.jsonmessage import JSONMessage
+            from tilecloud.store.sqs import SQSTileStore  # pylint: disable=import-outside-toplevel
+            import boto.sqs  # pylint: disable=import-outside-toplevel,import-error
+            from boto.sqs.jsonmessage import (  # pylint: disable=import-outside-toplevel,import-error
+                JSONMessage,
+            )
 
             region_name, queue_name = name[6:].split("/", 1)
             connection = boto.sqs.connect_to_region(region_name)
             queue = connection.create_queue(queue_name)
             queue.set_message_class(JSONMessage)
             return SQSTileStore(queue)
         if name.startswith("redis://"):
-            from tilecloud.store.redis import RedisTileStore
+            from tilecloud.store.redis import RedisTileStore  # pylint: disable=import-outside-toplevel
 
             return RedisTileStore(name)
         _, ext = os.path.splitext(name)
         if ext == ".bsddb":
-            import bsddb
-            from tilecloud.store.bsddb import BSDDBTileStore
+            import bsddb  # pylint: disable=import-outside-toplevel,import-error
+            from tilecloud.store.bsddb import BSDDBTileStore  # pylint: disable=import-outside-toplevel
 
             return BSDDBTileStore(bsddb.hashopen(name))
         if ext == ".mbtiles":
-            import sqlite3
-            from tilecloud.store.mbtiles import MBTilesTileStore
+            import sqlite3  # pylint: disable=import-outside-toplevel
+            from tilecloud.store.mbtiles import MBTilesTileStore  # pylint: disable=import-outside-toplevel
 
             return MBTilesTileStore(sqlite3.connect(name))
         if ext == ".zip":
-            import zipfile
-            from tilecloud.store.zip import ZipTileStore
+            import zipfile  # pylint: disable=import-outside-toplevel
+            from tilecloud.store.zip import ZipTileStore  # pylint: disable=import-outside-toplevel
```

Comment:
Seems you could consider putting `# pylint: disable=import-outside-toplevel` on top of file
ex: https://github.com/camptocamp/c2cgeoportal/blob/master/admin/tests/test_edit_url.py#L1

Reply:
+1 Done :-)